lib/EntryManager/EntryForm.js

```diff
@@ -113,11 +121,12 @@ class EntryForm extends React.Component {
             {selectedEntry &&
               <ConfirmationModal
                 open={confirmDelete}
-                heading={`Delete ${this.props.entryLabel}?`}
-                message={(<span><strong>{selectedEntry[this.props.nameKey] || `Untitled ${this.props.entryLabel}`}</strong> will be <strong>removed</strong>.</span>)}
+                heading={deleteButtonText}
```

Comment:
We lost the `<strong>` formatting here.

Reply:
So should I split up the translation to the sentence, the entry and the verb and apply the `<strong>` formatting in the js code?

Reply:
Yeah that's how we're moving forward for the time being. It's not pretty, but it's the best solution we've got to situations like this.

Reply:
As you also read on #stripes Slack channel we probably will have a safeFormattedHTMLMessage component which will allows using HTML elements in translations and will sanitize them. I would recommend to move the `<strong>` formatter in the translation and use the mentioned function when it will be available. Thoughts?